spec/locales-currencies-tz.html

```diff
@@ -205,41 +205,41 @@ <h1>DefaultTimeZone ( )</h1>
     <h1>Measurement Unit Identifiers</h1>
 
     <p>
-      The ECMAScript 2022 Internationalization API Specification identifies measurement units using a <em>core unit identifier</em> as defined by <a href="https://unicode.org/reports/tr35/tr35-general.html#Unit_Elements">Unicode Technical Standard #35, Part 2, Section 6</a>. Their canonical form is a string containing all lowercase letters with zero or more hyphens.
+      The ECMAScript 2022 Internationalization API Specification identifies measurement units using a <em>core unit identifier</em> (or equivalently <em>core unit ID</em>) as defined by <a href="https://www.unicode.org/reports/tr35/tr35-general.html#Unit_Identifiers">Unicode Technical Standard #35, Part 2, Section 6.2</a>. Their canonical form is a string containing all lowercase letters with zero or more hyphens.
     </p>
 
     <p>
-      Only a limited set of core unit identifiers are allowed.
-      An illegal core unit identifier results in a RangeError.
+      Only a limited set of core unit identifiers are supported.
+      Attempting to use an unsupported core unit identifier results in a RangeError.
     </p>
 
     <emu-clause id="sec-iswellformedunitidentifier" aoid="IsWellFormedUnitIdentifier">
       <h1>IsWellFormedUnitIdentifier ( _unitIdentifier_ )</h1>
 
       <p>
-        The IsWellFormedUnitIdentifier abstract operation verifies that the _unitIdentifier_ argument (which must be a String value) represents a well-formed core unit identifier as defined in <a href="https://unicode.org/reports/tr35/tr35-general.html#Unit_Elements">UTS #35, Part 2, Section 6</a>. In addition to obeying the UTS #35 core unit identifier syntax, _unitIdentifier_ must be one of the identifiers sanctioned by UTS #35 or be a compound unit composed of two sanctioned simple units. The following steps are taken:
+        The IsWellFormedUnitIdentifier abstract operation verifies that the _unitIdentifier_ argument (which must be a String value) represents a well-formed UTS #35 core unit identifier that is either a supported simple unit or a complex unit formed by division of two supported single units. The following steps are taken:
       </p>
 
       <emu-alg>
-        1. If the result of IsSanctionedSimpleUnitIdentifier(_unitIdentifier_) is *true*, then
+        1. If ! IsSanctionedSimpleUnitIdentifier(_unitIdentifier_) is *true*, then
           1. Return *true*.
-        1. If the substring *"-per-"* does not occur exactly once in _unitIdentifier_, then
+        1. Let _i_ be ! StringIndexOf(_unitIdentifier_, *"-per-"*, 0).
+        1. If _i_ is -1 or ! StringIndexOf(_unitIdentifier_, *"-per-"*, _i_ + 1) is not -1, then
```

Comment:
These editorial algorithm changes are gratuitous, but they do fit in with the rest of the PR.